tests/test_common.py

```diff
@@ -106,15 +106,6 @@ def test_slugify():
     assert common.slugify("あいうえお") == ""
```

Comment:
I dont understand this bit

Reply:
This is just testing that the slugify function (which makes a string safe to be used in URLs and filenames) works with multi byte Unicode characters, as historically they have caused trouble. So this just checks they are properly removed.

Reply:
```suggestion
    # Multi-byte unicode characters are removed.
    assert common.slugify("あいうえお") == ""
```